src/Json.php

```
@@ -26,14 +29,37 @@
         $jsonForSchema = json_encode($ini, JSON_THROW_ON_ERROR);
 
         $schema = Generator::fromJson($jsonForSchema, [
             'description' => sprintf('Generated from %s', basename($iniFile)),
         ]);
-        $this->schema = json_encode(json_decode($schema), JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES) . PHP_EOL;
+
+        // json_decode can return null/false/array/object, json_encode expects array/object
+        $decodedSchema = json_decode($schema);
+        /** @psalm-suppress ImpureFunctionCall */
+        if ($decodedSchema === null && json_last_error() !== JSON_ERROR_NONE) {
+             RuntimeException('Failed to decode generated schema JSON'); // @codeCoverageIgnore
```

PHPStan (2.0.3)

Function RuntimeException not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)